controller/appcontroller.go

```diff
@@ -359,17 +359,13 @@ func (ctrl *ApplicationController) handleObjectUpdated(managedByApp map[string]b
 			level = CompareWithRecent
 		}
 
-		// Additional check for debug level so we don't need to evaluate the
-		// format string in case of non-debug scenarios
-		if log.GetLevel() >= log.DebugLevel {
-			var resKey string
-			if ref.Namespace != "" {
-				resKey = ref.Namespace + "/" + ref.Name
-			} else {
-				resKey = "(cluster-scoped)/" + ref.Name
-			}
-			log.Debugf("Refreshing app %s for change in cluster of object %s of type %s/%s", appKey, resKey, ref.APIVersion, ref.Kind)
+		var resKey string
+		if ref.Namespace != "" {
+			resKey = ref.Namespace + "/" + ref.Name
+		} else {
+			resKey = "(cluster-scoped)/" + ref.Name
 		}
+		log.Infof("Refreshing app %s for change in cluster of object %s of type %s/%s", appKey, resKey, ref.APIVersion, ref.Kind)
```

Comment:
Lets try to get cluster name printed in the logs to know which cluster the app is located

Reply:
I think the comment is more related to https://github.com/agaudreault-jive/argo-cd/blob/704c880ac47c8ace6b04b6d4123a164befc87687/controller/cache/cache.go#L501 where we are still in a specific cluster context.

I can add the cluster server with 
`log.Debugf("Ignoring change in cluster %s of object %s of type %s/%s", clusterCache.GetClusterInfo().Server, resKey, ref.APIVersion, ref.Kind)`

However, I think it would be better if we would be able to use a `clusterCache.GetLogger()` method that would return a log Entry with the current context set on it. This would require changes to the gitops-engine code.

I am not sure that adding the cluster server will provide much value for this log, but I will add it and set a context with WithFields for both logs so we do not need to parse the message 👍